### ISSUE TITLE
Exception in delivering result of invoking 'email/verifySettings'

### ISSUE THREAD

**Issue by aaronjudd** [reactioncommerce/reaction#2464]
Open email settings in dashboard, update settings, close the dashboard panel, view browser console:

```Exception in delivering result of invoking 'email/verifySettings': Error: Tyring set data after component(Container(EmailConfig)) has unmounted.
    at onData (http://localhost:3000/packages/modules.js?hash=324c20e0822d5dd5972b7d40d96a63c0b55798e8:18970:21)
    at http://localhost:3000/app/app.js?hash=35ec44c0610a5ab62f80750be4a4e4201baed3fd:72691:18
    at MethodInvoker._callback (http://localhost:3000/packages/meteor.js?hash=27829e936d09beae3149ecfbf3332c42ccb1596f:1105:22)
    at MethodInvoker._maybeInvokeCallback (http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:3678:12)
    at MethodInvoker.receiveResult (http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:3698:10)
    at Connection._livedata_result (http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:4816:9)
    at onMessage (http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:3525:206)
    at http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:2908:9
    at Array.forEach (native)
    at Function._.each._.forEach (http://localhost:3000/packages/underscore.js?hash=cde485f60699ff9aced3305f70189e39c665183c:149:11)
```

Node: 7.5.0
NPM: 5.0.0
Meteor Node: 4.8.3
Meteor NPM: 4.6.1
Reaction CLI: 0.10.0
Reaction: 1.3.0
Reaction branch: updated-packages-1.3.1
Docker: 17.03.1-ce

**Comment by kieckhafer** [comment]
I've narrowed this down to the component re-rendering when we close the side-panel, and our `Meteor.call` [here](https://github.com/reactioncommerce/reaction/blob/development/imports/plugins/core/email/client/containers/emailConfig.js#L27) sending data back as the component is unmounted. I'm not sure why it's re-rendering...

There are similar issues in the `react-komposer` github issues list, I've left a comment on an issue to see if there has been a solution found.

**Comment by zenweasel** [comment]
Assigning to @kieckhafer as he seems to understand the issue. Sorry dude.

**Comment by kieckhafer** [comment]
I believe I fixed this in pr #2416, @zenweasel can you confirm?

**Comment by zenweasel** [comment]
@kieckhafer No, still happening.

**Comment by zenweasel** [comment]
@kieckhafer Let me assign this to someone else.

**Comment by awadesmoline** [comment]
@kieckhafer I also confirmed this to be caused by `react-komposer`. It calls the method when the component is unmounting, which tries to set the data for the component after it has unmounted already.
Please check if the fix [here](https://github.com/reactioncommerce/reaction/pull/2512) can be okay for now.

**Comment by awadesmoline** [comment]
Closed via #2512
